concent_api/concent_api/constants.py

```diff
@@ -43,6 +44,15 @@
             url(r'^gatekeeper/', include(gatekeeper.urls, namespace = 'gatekeeper')),
         ],
     }),
+
+    ("verifier", {
+        "required_django_apps": [
+            "verifier",
+        ],
+        "url_patterns": [
+            url(r'^verifier/', include(verifier.urls, namespace='verifier')),
```

Comment:
Verifier is not supposed to have any endpoints.

Reply:
:+1: